tests/integ/test_local_mode.py

```diff
@@ -261,6 +261,21 @@ def test_local_mode_serving_from_local_model(tmpdir, sagemaker_local_session, mx
                 predictor.delete_endpoint()
 
 
+def test_local_deploy_with_update_endpoint(tmpdir, sagemaker_local_session, mxnet_model):
```

Comment:
You can remove this integration test. I don't think this is necessary, as the unit test already catches this use case.

If you want to keep the test, let's assert on the exception message and catch the specific exception as well.